src/pages/Calendar.css

```
@@ -62,12 +62,88 @@
+
+/* 검색창은 남는 폭을 다 먹되, 드롭다운이 밀려나지 않게 최소폭을 준다. */
+.calendar-filters ion-searchbar.calendar-filters__search {
+  flex: 1 1 220px;
+  min-width: 180px;
+  padding: 0;
+  --background: var(--surface-field);
+  --box-shadow: none;
+  --border-radius: var(--radius-sm);
+  --placeholder-color: var(--text-muted);
+  --color: var(--text-body);
+  height: 38px;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)